app/views/common/500.html.erb

```diff
@@ -1,21 +1,20 @@
 <%= alert :header => _("Oops, we're sorry but something went wrong"), :class => 'alert-danger',
   :actions => link_to(_('Back'), main_app.root_path, :class => 'btn btn-default'),
-  :text => h(exception.message),
+  :text => h(exception_message),
   :close => false %>
 
 <div class="pull-right">
   <%= documentation_button("7.3GettingHelp")%>
 </div>
+
 <p id="message">
   <%= _("If you feel this is an error with Foreman itself, please open a new issue with") %>
-  <%= link_to _("Foreman ticketing system"), "http://theforeman.org/projects/foreman/issues", :rel => "external" %>,
-  <%= _("You would probably need to attach the") %>
-  <%= link_to_function(_("Full trace"), "$('#backtrace').toggle()")%>
-  <%= _(", relevant log entries, and it is highly recommended to also attach the foreman-debug output.") %>
+  <%= link_to _("Foreman ticketing system"), "https://projects.theforeman.org/", :rel => "external" %>,
+  <%= _("Please include in your report the full error log that can be acquired by running: ") %>
+  <strong> foreman-rake errors:fetch_log request_id=<%= request_id %></strong>
+  <%= _(" and it is highly recommended to also attach the foreman-debug output.") %>
+  <%= _("The error code is ") %>
+  <%= exception_code %>
```

Comment:
Would it make sense to link this to some page for the specific error code? (currently we have https://projects.theforeman.org/projects/foreman/wiki/ErrorCodes that has a page for many error codes with possible resolutions, but it is likely very out of date)

Reply:
I don't think it's a good idea because there is only several pages created for bugs. Maybe it will be better only to link the page with list of ErrorCodes. Do you think it will be good resolution for that?

Reply:
I think it's ok to link to this page since it's a wiki which means people can create pages. Since the codes are generated dynamically, it's not going to be possible create an exhaustive list. Having a partial list is better than nothing.

Reply:
Linking our error codes is a good idea actually, it's not too out of date and I occasionally regenerate it. If you put this to the release TODO list, feel free to bug me every time :-)